scala-api/pom.xml

```diff
@@ -46,6 +46,19 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>com.cloudera.livy</groupId>
```

Comment:
livy-core is added as a dependency twice here. One with <type>test-jar</type> andother withouth.
Is this intentional?